bower.json

```diff
@@ -2,20 +2,21 @@
   "name": "jupyter-notebook-deps",
   "version": "0.0.1",
   "dependencies": {
-    "backbone": "components/backbone#~1.2",
-    "bootstrap": "components/bootstrap#~3.3",
-    "bootstrap-tour": "0.9.0",
-    "codemirror": "~5.5",
     "es6-promise": "~1.0",
     "font-awesome": "components/font-awesome#~4.2.0",
+    "requirejs": "~2.1",
     "google-caja": "5669",
+    "MathJax": "components/MathJax#~2.5",
+    "bootstrap": "components/bootstrap#~3.3",
+    "bootstrap-tour": "0.9.0",
     "jquery": "components/jquery#~2.0",
     "jquery-ui": "components/jqueryui#~1.10",
+    
```

Comment:
JSON doesn't support comments, so I'm leaving one here in GitHub.   The following packages can be removed from bower and the setupbase check now, but I left them here for backwards compatibility...  In case any users are loading them in via requirejs.